services/api-server/src/simcore_service_api_server/services/log_streaming.py

```diff
@@ -0,0 +1,135 @@
+import asyncio
+from asyncio import Queue
+from typing import AsyncIterable, Awaitable, Callable, Final
+
+from models_library.rabbitmq_messages import LoggerRabbitMessage
+from models_library.users import UserID
+from pydantic import PositiveInt
+from servicelib.rabbitmq import RabbitMQClient
+
+from ..models.schemas.jobs import JobID, JobLog
+from .director_v2 import DirectorV2Api
+
+_NEW_LINE: Final[str] = "\n"
+_SLEEP_SECONDS_BEFORE_CHECK_JOB_STATUS: Final[PositiveInt] = 10
```

Comment:
Here I really suggest making this configurable with an env-var. As a design principle, all hard-coded numbers should be env-vars, then we can really change them reliably and fast. We are comfortable (or, well, we simply need to be) handling many env-vars.